### ISSUE TITLE
Nitter RSS/list does not show retweets.

### ISSUE THREAD

**Issue by Giger22** [zedeus/nitter#889]
You need to go to a specific account page to see retweets. In a list view there are no retweets visible.
 
https://nitter.net/Everton,premierleague,adrianjclarke

**Comment by pin-grid-array** [comment]
It seems there have been intermittent problems with multi-Twitter-user feeds. See https://github.com/zedeus/nitter/issues/852 and https://github.com/zedeus/nitter/issues/855, for example.

**Comment by zedeus** [comment]
This is an issue with Twitter's GraphQL API for search, which became necessary to use after they closed off access to the old API. It seems there's nothing Nitter can do about this, but I'll keep this issue open.

**Comment by Giger22** [comment]
Can't multi feed tweets be scraped directly from a timeline?

**Comment by zedeus** [comment]
Not sure what you mean by that. Multi-feed timelines use search and cannot be enhanced further, they were meant as a quick temporary solution until accounts are implemented.

**Comment by Giger22** [comment]
I meant to use a multi-feed as list function on twitter/RSS without having an account on twitter. Obviously a lack of retweets in a multi-feed view makes this a poor option. 
I think fixing this problem would be easy. If you can show a retweet in a normal timeline why not in a multi-feed which serves as an alternative to having an account on twitter if you want to follow a lot of accounts separated in lists.

**Comment by zedeus** [comment]
Multi-feed as list? I don't know what you mean, Twitter has no way to retrieve all tweets and retweets from a list of accounts without making separate requests for all of them and combining them. You cannot create a Twitter list without having an account. As I said, the reason retweets aren't being shown anymore is the new endpoint doesn't support the `include:nativeretweets` feature that the old one did. You say it's easy to fix but you aren't suggesting any solutions.

**Comment by Giger22** [comment]
I was talking about adding retweets from a single scraped account page that shows retweets to a multi-feed view.

**Comment by zedeus** [comment]
Right, you'd have to fetch every account individually, in which case performing a search to get the multi-feed makes no sense since you'd then get all the tweets anyway. This is extremely resource intensive and not feasible for an instance with many users. There is an alternative solution that will fix this with the introduction of an account system, but it won't be ready for a while.

**Comment by ares-osint** [comment]
https://github.com/zedeus/nitter/issues/944#issuecomment-1638682073
